src/company/team-sum-up/team-sum-up.service.ts

```diff
@@ -200,16 +199,16 @@ export class TeamSumUpService extends CrudService<
       const todayDate = moment.utc().format('YYYY-MM-DD');
       const result = await prisma.$transaction(
         async (client: CompanyPrismaClient) => {
-          const notifictionExist = await client.notifications.findFirst({
+          const notificationExist = await client.notifications.findFirst({
             where: {
               sentAt: {
                 gte: new Date(`${todayDate}T00:00:00.000Z`),
                 lt: new Date(`${todayDate}T23:59:59.999Z`),
               },
             },
           });
-          if (notifictionExist) {
-            throw new ConflictException('notification exist already!');
+          if (notificationExist) {
+            return null;
```

Comment:
you can just return instead of null
```suggestion
            return;
```